packages/e2e-tests/specs/editor/plugins/__snapshots__/cpt-locking.test.js.snap

```diff
@@ -41,7 +41,7 @@ exports[`cpt locking template_lock all should not error when deleting the cotent
 `;
 
 exports[`cpt locking template_lock all unlocked group should allow blocks to be moved 1`] = `
-"<!-- wp:group {\\"templateLock\\":false} -->
+"<!-- wp:group {\\"templateLock\\":false,\\"layout\\":{\\"type\\":\\"constrained\\"}} -->
```

Comment:
Oh interesting, so this changed again without the test being altered in any way?

Reply:
There was a related change above: https://github.com/WordPress/gutenberg/pull/43496/files#diff-8ea13eb3838ceaa967fbed75698db9216cea3d86ebd584423730824556c7b974R72

Thanks for pointing it out however. Maybe we don't need the change anymore.